dataset_loaders/videos/davis.py

```diff
@@ -55,12 +55,19 @@ def filenames(self):
     def __init__(self,
                  which_set='train',
                  threshold_masks=False,
+                 optical_flow_type=None,
                  split=.75,
                  *args, **kwargs):
```

Comment:
Please add a description of the parameters (some of them you can copy from other datasets)